tests/models/test_linear_network.py

```diff
@@ -25,7 +26,8 @@ def mocktrain(self):
         monkeypatch.setattr(LinearNetwork, 'train', mocktrain)
 
         model = LinearNetwork(data_folder=tmp_path, layer_sizes=layer_sizes,
-                              dropout=dropout, include_pred_month=include_pred_month)
+                              dropout=dropout, include_pred_month=include_pred_month,
+                              surrounding_pixels=surrounding_pixels)
```

Comment:
How big will the number of features be when we have a `surrounding_pixels` argument of 1?

with 3 variables:
```
( (12 months x 3) x 2 latlon shifts (surrounding_pixels =1) )  + 12 one_hot_months 
```

Reply:
So if `surrounding pixels == 1`, historical gets `9x` more features. So I think it would be:

`12 months x 3 variables x 9 pixels) + 12 one hot months`